lib/search.js

```diff
@@ -63,7 +63,7 @@ function search(graphUri, criteria, offset, limit, user, explorer = false) {
                     result.url = result.uri.toString()
 		}
 
-                result.triplestore = result.uri.toString().startsWith(config.get('databasePrefix')+'public/')?'public':'private'
+                result.triplestore = result.uri.toString().startsWith(config.get('databasePrefix')+'public/')?'public':result.uri.toString().startsWith(config.get('databasePrefix')+'user/')?'private':'remote'
```

Comment:
This works, but the ternary operators and the `?` and `:` in the URL make it hard to read. It could probably benefit from being an if/else clause.